### DISCUSSION TITLE
JKBMS B2A24S15P & v1.0.20230531 Never switches to FLOAT

### DISCUSSION THREAD

**Discussion by BipedalPrimate** [Louisvdw/dbus-serialbattery#926] [Q&A] [answered]
As I understand it, the following parameters in config.ini (full file attached as config.ini.txt below) determine when to switch to Float:

MAX_CELL_VOLTAGE   = 3.450
FLOAT_CELL_VOLTAGE = 3.350
CVCM_ENABLE = False     (Started with this set to True but never achieved 55.2v because of initially unbalanced pack - now balanced)
CELL_VOLTAGE_DIFF_KEEP_MAX_VOLTAGE_UNTIL = 0.010
CELL_VOLTAGE_DIFF_TO_RESET_VOLTAGE_LIMIT = 0.160
MAX_VOLTAGE_TIME_SEC = 600    (note that I have tried setting this to 1 sec)
SOC_LEVEL_TO_RESET_VOLTAGE_LIMIT = 90

It appears that all of these conditions have been met on a number of occasions yet Float is never achieved.

Am I missing something?  If so, what?

![image](https://github.com/Louisvdw/dbus-serialbattery/assets/142499479/38103bd6-f582-4ef9-b8f3-e80ba28855fb)

![image](https://github.com/Louisvdw/dbus-serialbattery/assets/142499479/0f16ca52-e602-485a-a850-e5c9ab970276)

![Screenshot_20240119-115125_BMS](https://github.com/Louisvdw/dbus-serialbattery/assets/142499479/4d8df7bd-4fed-4d81-b409-600d059976e6)

![Screenshot_20240119-115430_BMS](https://github.com/Louisvdw/dbus-serialbattery/assets/142499479/60c6572c-32f6-4a28-8346-d8296435121d)
![Screenshot_20240119-115510_BMS](https://github.com/Louisvdw/dbus-serialbattery/assets/142499479/d178b502-e185-4e34-acd6-c3327a9b25b9)
![Screenshot_20240119-115627_BMS](https://github.com/Louisvdw/dbus-serialbattery/assets/142499479/b385b219-39f8-46e8-9e34-255c313b5452)

[config.ini.txt](https://github.com/Louisvdw/dbus-serialbattery/files/13983215/config.ini.txt)

**Comment by BipedalPrimate** [comment]
Incorrectly submitted in Discussions

**Comment by mr-manuel** [comment, marked as answer]
Duplicate of https://github.com/Louisvdw/dbus-serialbattery/issues/927
